gordo_components/watchman/endpoints_status.py

```diff
@@ -95,8 +95,8 @@ def __init__(
                     "`listen_to_kubernetes`==True"
                 )
             # If we are listening to kubernetes events
-            # we only do a "manual" update of every endpoint every 20 min
-            self.happy_update_model_interval = 1200
+            # we only do a "manual" update of every endpoint every hour
+            self.happy_update_model_interval = 3600
```

Comment:
To have less load on the cluster when there are 6000 models. Since we are listening to changes on the cluster this actually only has one effect, and that is if we happened to get exactly [here](https://github.com/equinor/gordo-components/blob/fe6793a1b211fd832da7f5ae7e148bbc3a5fa03a/gordo_components/watchman/gordo_k8s_interface.py#L113), AND there is a model deleted before we manage to get to the `watch` again. 

The alternative, which might be better, could be to pretty much implement a reflector. But that is a  tad more work:-/

Reply:
Most excellent. I can get on board with that logic. :thought_balloon: